src/test/java/io/strimzi/kafka/bridge/mqtt/MqttBridgetIT.java

```diff
@@ -0,0 +1,238 @@
+    private static MqttServer mqttBridge;
+    private static StrimziKafkaContainer kafkaContainer;
+    private String mqttClientId;
+
+
+    /**
+     * Start the kafka Cluster
+     * Start the MQTT bridge before all tests
+     */
+    @BeforeAll
+    public static void beforeAll() throws InterruptedException {
+        String kafkaBootstrapServers = null;
+        try {
+            kafkaContainer = new StrimziKafkaContainer();
+            kafkaContainer.waitForRunning();
+            kafkaContainer.start();
+            kafkaBootstrapServers = kafkaContainer.getBootstrapServers();
+        } catch (Exception e) {
+            logger.error("Exception occurred", e);
+            throw e;
+        }
+
+        // instantiate the kafka consumer client
+        kafkaConsumerClient = new KafkaConsumer<>(
+                ImmutableMap.of(
+                        ConsumerConfig.BOOTSTRAP_SERVERS_CONFIG, kafkaBootstrapServers,
+                        ConsumerConfig.GROUP_ID_CONFIG, "gid-" + UUID.randomUUID(),
+                        ConsumerConfig.AUTO_OFFSET_RESET_CONFIG, OffsetResetStrategy.EARLIEST.name().toLowerCase(Locale.ROOT)
+                ),
+                new StringDeserializer(),
+                new StringDeserializer()
+        );
+
+        // consumer client subscribes to the kafka topic
+        kafkaConsumerClient.subscribe(Collections.singletonList(KAFKA_TOPIC));
+
+        // prepare the configuration for the bridge
+        BridgeConfig bridgeConfig = BridgeConfig.fromMap(
+                ImmutableMap.of(
+                        BridgeConfig.BRIDGE_ID, "my-bridge",
+                        KafkaConfig.KAFKA_CONFIG_PREFIX + ConsumerConfig.BOOTSTRAP_SERVERS_CONFIG, kafkaBootstrapServers,
+                        MqttConfig.MQTT_HOST, MQTT_SERVER_HOST,
+                        MqttConfig.MQTT_PORT, MQTT_SERVER_PORT
+                ));
+
+        // prepare the mapping rules
+        String mappingRulesPath = Objects.requireNonNull(MqttBridgetIT.class.getClassLoader().getResource("mapping-rules-regex.json")).getPath();
+        MappingRulesLoader.getInstance().init(mappingRulesPath);
+
+        // start the MQTT bridge
+        CountDownLatch latch = new CountDownLatch(1);
+
+        EventLoopGroup bossGroup = new NioEventLoopGroup();
+        EventLoopGroup workerGroup = new NioEventLoopGroup();
+
+        mqttBridge = new MqttServer(bridgeConfig, bossGroup, workerGroup, ChannelOption.SO_KEEPALIVE);
+        mqttBridge.start();
+
+        latch.countDown();
+        latch.await();
+    }
+
+    /**
+     * Randomly generate a new client id before each test
+     */
+    @BeforeEach
+    public void generateClientId() {
+        mqttClientId = "mqtt-client-" + new Random().nextInt(20);
+    }
+
+    /**
+     * Test the client's connection to the Bridge
+     */
+    @Test
+    public void testConnection() throws MqttException {
+        try (MqttClient client = new MqttClient(MQTT_SERVER_URI, mqttClientId)) {
```

Comment:
... and call `generateClientId()` directly here instead of referencing `mqttClientId`.